code/modules/power/apc/apc_contextual_tips.dm

```diff
@@ -20,7 +20,7 @@
 			context[SCREENTIP_CONTEXT_LMB] = "Open the cover"
 		else if ((opened == APC_COVER_OPENED && has_electronics == APC_ELECTRONICS_SECURED) && !(machine_stat & BROKEN))
 			context[SCREENTIP_CONTEXT_LMB] = "Close and lock"
-		else if (machine_stat & BROKEN|(machine_stat & EMAGGED| malfhack))
+		else if (machine_stat & (BROKEN|EMAGGED|malfhack))
```

Comment:
This looks like an error for another reason, `malfhack` is a boolean

Reply:
Converted it to a seperate check via https://github.com/tgstation/tgstation/pull/92609/commits/3f84f5f2d28eec4384be273a7c588668cfcaf43a